packages/wayland_info.rb

```diff
@@ -0,0 +1,38 @@
+require 'package'
+
+class Wayland_info < Package
+  description 'wayland-info is a utility for displaying information about the Wayland protocols supported by a Wayland compositor.'
+  homepage 'https://gitlab.freedesktop.org/wayland/wayland-utils'
+  @commit = '3fec240801442ae229112d35e070ea71eac06d0b'
+  version "1.1-#{@commit[0...7]}"
```

Comment:
Can we change `@commit` to `@_ver` so there is some consistency for extracting the version?

Reply:
Done.